connectors/database/mysql.py

```
@@ -21,37 +20,39 @@ def __init__(self):
-        
+    def create_engine(self, hostname, username, password, port, database, connection_name=None, connection_type=None,
+                      engine="MySQL", schema="defaultdb", **kwargs):
+        return super().create_engine(engine, hostname, username, password, port, database, connection_name=None,
+                                     connection_type=None, schema=schema)
+
     def get_metadata(self, *args, **kwargs):
         auth_details = kwargs
         return super().get_metadata(**auth_details)
-    
+
     def get_metadata(self):
```

Ruff (0.11.9)

Redefinition of unused `get_metadata` from line 34

(F811)